### PR TITLE
Update roadmap doc with latest releases

### PR DESCRIPTION
**Pull request by mattyw**

### CODE REVIEW

**Comment by randall** [comment]
[approve ci]

**Comment by masaori335** [comment]
[approve ci]

File: doc/release-notes/roadmap.en.rst [hunk 1]
```diff
@@ -27,7 +27,7 @@ ATS Release Roadmap
 
 Apache Traffic Server is currently on a two year major release cycle. At any
 given time, we guarantee to have two major versions supported. Currently,
-this means v7.1.x and v8.0.x, with a possible v8.1.x coming.
+this means v8.1.x and v9.2.x, with a possible v10.x.x coming.
```
Comment (masaori335) [inline doc/release-notes/roadmap.en.rst:30]: `v8.x` and `v9.x` is better? The current latest version is 9.1.2, but 9.2.0 is coming soon.

Comment (bryancall) [reply doc/release-notes/roadmap.en.rst:30]: The recommendation from our PR and issue scrub was to state something that doesn't need to be updated.  "Please refer to our download page to see the current supported versions."  and link to the download page.

**Comment by mattyw** [comment]
@masaori335 @bryancall Updated. PTAL

**Comment by masaori335** [comment]
@mattyw Thanks for correcting docs.

**Comment by zwoop** [comment]
Cherry-picked to v9.2.x

**Pull request merged**